requirements/ci.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
We should still be compiling requirements with python 3.8 so that we don't upgrade past something we need to run on 3.8 still.

Reply:
Okay I'll update according to it.

Reply:
@feanil  have updated